#! /bin/sh /usr/share/dpatch/dpatch-run
## 72_deprecated_hal_key.dpatch by  <martin.pitt@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: info.bus hal property is deprecated with 0.5.11, use info.subsystem;
## DP: http://gphoto.svn.sourceforge.net/viewvc/gphoto/trunk/libgphoto2/packaging/generic/print-camera-list.c?r1=10735&r2=10967
## DP: https://bugs.launchpad.net/205417

@DPATCH@
diff -urNad libgphoto2-2.4.0~/packaging/generic/print-camera-list.c libgphoto2-2.4.0/packaging/generic/print-camera-list.c
--- libgphoto2-2.4.0~/packaging/generic/print-camera-list.c	2007-07-27 02:35:28.000000000 +0200
+++ libgphoto2-2.4.0/packaging/generic/print-camera-list.c	2008-03-25 16:29:31.000000000 +0100
@@ -798,7 +798,7 @@
 	print_version_comment(stdout, "    | ", "\n", "<!--+\n", "    +-->\n");
 	printf("<deviceinfo version=\"0.2\">\n");
 	printf(" <device>\n");
-	printf("  <match key=\"info.bus\" string=\"usb\">\n");
+	printf("  <match key=\"info.subsystem\" string=\"usb\">\n");
 	return 0;
 }
 
@@ -917,7 +917,7 @@
 	print_version_comment(stdout, "    | ", "\n", "<!--+\n", "    +-->\n");
 	printf("<deviceinfo version=\"0.2\">\n");
 	printf(" <device>\n");
-	printf("  <match key=\"info.bus\" string=\"usb\">\n");
+	printf("  <match key=\"info.subsystem\" string=\"usb\">\n");
 	return 0;
 }
 
